.travis.yml

```diff
@@ -1,8 +1,16 @@
 language: python
 
-python:
-  - "2.7"
-  - "3.6"
+matrix:
+    include:
+        - python: 2.7
+          env: NUMPY_VERSION=1.12
+
+        - python: 3.6
+          env: NUMPY_VERSION=1.12 NODEPS=true
```

Comment:
Why do we have two 3.6 tests now?

Reply:
This one tests with only the minimum dependencies installed. I thought it couldn't hurt since we tend to test with our development environment where everything is installed, so errors like unfenced imports aren't caught.

Reply:
I'd say that is overly ambitious, especially as we want these to run asap and also to give easy to read errors, adding two slightly different will confuse most people

Reply:
Runtime isn't an issue since they run in parallel and the first failure determines the overall result. I can also make the variable more self-explanatory, like `MINIMAL_DEPENDENCIES` or so.

That said, I don't really mind removing the setting. But then I'll probably add an option `COVERALLS` so that only one of the builds runs `coveralls`.

Reply:
Agree that only one needs to run coveralls. Overall I think "micro-managing" like this is somewhat risky since it makes this stuff harder to follow